docs/developer/manual-localdev.md

```diff
@@ -0,0 +1,75 @@
+# Manual Local Development
+
+Here, we give an overview of a development setup for Armada that gives users full control over the Armada components and dependencies.
+
+Before starting, please ensure you have installed [Go](https://go.dev/doc/install) (version 1.20 or later), gcc (for Windows, see, e.g., [tdm-gcc](https://jmeubank.github.io/tdm-gcc/)), [mage](https://magefile.org/), [docker](https://docs.docker.com/get-docker/), [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl), and, if you need to compile `.proto` files, [protoc](https://github.com/protocolbuffers/protobuf/releases).
+
+For a full lust of mage commands, run `mage -l`.
+
+## Setup
+
+### Note for Arm/M1 Mac Users
```

Comment:
Can you file a ticket and maybe say we welcome contributors to help with a windows development environment?

Reply:
ie wsl or whatever.

Reply:
Sounds like a plan :)

Reply:
Adding to developer.md instead of here.